### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mishagitcode**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class CinemaHall:
+
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list,
```
Comment (EdAlekseiev) [inline app/cinema/hall.py:12]: You must specify the inner type of list

Comment (EdAlekseiev) [inline app/cinema/hall.py:12]:
```suggestion
            customers: list[Customer],
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list,
+        hall_number: int,
+        cleaner: str,
+        movie: str
+) -> None:
+
+    cus_ins_ls = []
```
Comment (ArsenPidhoretskyi) [inline app/main.py:15]: cus_ins_ls
What this variable means?

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — APPROVED** [review]
Looks good

**Pull request closed without merging**